data-prepper-logstash-configuration/src/test/java/org/opensearch/dataprepper/logstash/mapping/LogstashMapperTest.java

```diff
@@ -0,0 +1,57 @@
+package org.opensearch.dataprepper.logstash.mapping;
+
+import com.amazon.dataprepper.model.configuration.PipelineModel;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.opensearch.dataprepper.logstash.exception.LogstashMappingException;
+import org.opensearch.dataprepper.logstash.model.LogstashConfiguration;
+import org.opensearch.dataprepper.logstash.model.LogstashPluginType;
+
+import java.util.Collections;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class LogstashMapperTest {
+
+    private LogstashMapper logstashMapper;
+
+    @BeforeEach
+    void createObjectUnderTest() {
+        logstashMapper = new LogstashMapper();
+    }
+
+    @Test
+    void mapPipeline_returns_pipeline_model() {
```

Comment:
Please create some tests to verify when the different plugin types are null. You have a null check for them in code, but we should test it.